pandas/io/json/table_schema.py

```diff
@@ -306,10 +306,9 @@ def parse_table_schema(json, precise_float):
         raise NotImplementedError('table="orient" can not yet read timezone '
                                   'data')
 
-    # No ISO constructor for Timedelta as of yet, so need to raise
```

Comment:
was this not hit in tests prior to this PR?

Reply:
Should have been covered by `test_read_json_table_orient_raises` through parametrization